pkgs/applications/version-management/git/default.nix

```diff
@@ -77,7 +77,9 @@ stdenv.mkDerivation (finalAttrs: {
 
   nativeBuildInputs = [ gettext perlPackages.perl makeWrapper pkg-config ]
     ++ lib.optionals withManual [ asciidoc texinfo xmlto docbook2x
-         docbook_xsl docbook_xml_dtd_45 libxslt ];
+         docbook_xsl docbook_xml_dtd_45 libxslt ]
+    ++ lib.optional (stdenv.buildPlatform != stdenv.hostPlatform) (
+         deterministic-uname.override { forPlatform = stdenv.hostPlatform; });
```

Comment:
This is an example of a package that can be fixed with a host-based `deterministic-uname`.

TODO: Remove this from the PR.